scheduler/kubernetes/upgrade-1.3.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: page
+title: "Upgrade Portworx on Kubernetes"
+keywords: portworx, container, Kubernetes, storage, Docker, k8s, flexvol, pv, persistent disk
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+
+>**IMPORTANT:**<br/>We do not recommend upgrading Portworx using [Kubernetes instructions](https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/) (e.g. via `kubectl set image ds/portworx portworx=portworx/XXXX:### -n kube-system`).
+
+<a name="oci-upgrade"></a>
+## Upgrading Portworx
+
+This guide describes the procedure to upgrade Portworx running as OCI container using our opensource project [talisman](https://github.com/portworx/talisman).
+
```

Comment:
Suggest removing the wording around opensource project and just say Talisman.

Reply:
done